lcc_comchain_base/tests/test_wallet.py

```diff
@@ -1,103 +1,104 @@
 import json
-
+from minimock import mock, Mock, restore
 from odoo.addons.component.tests.common import TransactionComponentCase
+from pyc3l import Pyc3l, Wallet
 
 
 class TestResWallet(TransactionComponentCase):
     def setUp(self):
         super().setUp()
 
-        self.ResUsers = self.env["res.users"]
         self.ResPartner = self.env["res.partner"]
         self.ResPartnerBackend = self.env["res.partner.backend"]
         self.ResAltCurrency = self.env["res.alt.currency"]
 
-        self.partner_roger = self.ResPartner.create({"name": "Roger"})
-        self.currency_A_unit_product = self.env.ref(
+        self.comchain_currency_unit_product = self.env.ref(
             "lcc_lokavaluto_app_connection.product_product_numeric_lcc"
         ).sudo()
-        self.currency_A = self.ResAltCurrency.create(
+
+    def _create_res_partner_backend(self, partner, currency, ident="98765"):
+        return self.ResPartnerBackend.create(
             {
-                "name": "Currency A",
-                "ident": "currencyA",
-                "active": True,
-                "engine": "comchain",
-                "currency_unit_product_id": self.currency_A_unit_product.id,
+                "name": f"comchain:{ident}",
+                "alt_currency_id": currency.id,
+                "partner_id": partner.id,
+                "comchain_id": ident,
+                "comchain_wallet": json.dumps("toto"),
+                "comchain_message_key": "tata"
```

Comment:
```suggestion
                "comchain_message_key": "bar"
```

c'est plus classique, surtout pour le jour où Lokavaluto devient internationalement connu et on se fait aider par des anglophones ;)